### PR TITLE
Formatting cleanups, fixed all warnings in markdowner.py and adapted GH Action to no longer allow warnings

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by olehermanse** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/505/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/505/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-505/output/_site/

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/506/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/506/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-506/output/_site/

**Comment by olehermanse** [comment]
@cf-bottom jenkins, please

**Review by nickanderson — APPROVED** [review]

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by nickanderson** [comment]
Did not cherry pick cleanly.

**Comment by olehermanse** [comment]
> Did not cherry pick cleanly.

Of course not, but I will still make it happen :)

**Comment by olehermanse** [comment]
Cherry-pick: https://github.com/cfengine/documentation/pull/3206

File: .github/workflows/markdowner.py [hunk 1]
```diff
@@ -68,11 +94,19 @@ def process_codeblock(lines, filename, lineno_start):
...
+        # we indented it to same level as opening backticks above:
+        if i == 0 and file_line:
+            lineno += 1
+            continue
+        # Don't consider empty lines for common indentation:
         if line == "":
             lineno += 1
             continue
         if line[len(prefix) :][0] != " ":
+            # Fond content without extra indentation -
```
[the remark this answers is not in the document]
Comment (olehermanse) [reply .github/workflows/markdowner.py:108]: Fixed in:
https://github.com/cfengine/documentation/pull/3207
https://github.com/cfengine/documentation/pull/3206

**Pull request merged**